cms/djangoapps/contentstore/views/course.py

```diff
@@ -97,6 +94,10 @@
     is_entrance_exams_enabled
 )
 
+from openedx.stanford.cms.djangoapps.contentstore.views.course import get_context_for_course_info_handler
+from openedx.stanford.cms.djangoapps.contentstore.views.course import get_context_for_settings_handler
+from openedx.stanford.cms.djangoapps.contentstore.views.course import send_test_enrollment_email
+
```

Comment:
This are now the only diff in the import section;
it should be obvious that it's ours only.